packages/ag-charts-enterprise/src/series/bullet/bulletModule.ts

```diff
@@ -24,6 +24,21 @@ export const BulletModule: _ModuleSupport.SeriesModule<'bullet'> = {
             },
         ],
     },
-    swapDefaultAxesCondition: (series) => series?.direction !== 'horizontal',
+    swapDefaultAxesCondition: (opts) => opts?.direction !== 'horizontal',
+    customDefaultAxesSwapper: (opts) => {
```

Comment:
I'm a bit confused why we need this extra level of control vs. the result of `swapDefaultAxesCondition()` - isn't the only different the order of declaration of the axes?